modules/bouncer_scriptworker/manifests/settings.pp

```diff
@@ -52,13 +62,21 @@
 
         bouncer_instances        => {
             "${bouncer_stage_instance_scope}"   => $bouncer_stage_instance_config,
+            "${bouncer_nazgul_stage_instance_scope}" => $bouncer_nazgul_stage_instance_config,
             'project:releng:bouncer:server:production' => {
                 api_root                  => 'https://bounceradmin.mozilla.com/api',
                 timeout_in_seconds        => 60,
                 username                  => 'ffxbld',
                 # TODO Split credentials
                 password                  => secret('tuxedo_password'),
             },
+            'project:releng:bouncer:server:production-nazgul' => {
+                api_root                  => 'https://bounceradmin-default.prod.mozaws.net/api',
+                timeout_in_seconds        => 60,
+                username                  => 'taskcluster',
+                # TODO Split credentials
```

Comment:
Do you need to address this todo?

Reply:
No, I kept the copy-paste from the old one. No action for now I suppose, until we're fully migrated to the new Nazgul.